target_chains/cosmwasm/tools/store/edge/test-contracts/injective_testnet-1.2.0.json

```diff
@@ -0,0 +1,34 @@
+{
+  "push-price-update": {
+    "status": "rejected",
```

Comment:
this seems bad?

Reply:
The error is expected. We have to get the injective team update the pyth contract address which they are using for the oracle.